frontend/src/model/inferencing/InferenceForm.tsx

```diff
@@ -98,7 +99,14 @@ export default function InferenceForm({
       {isReadOnly ? (
         <ReadOnlyAnswer value={formData.port.toString()} />
       ) : (
-        <TextField required size='small' value={formData.port} onChange={handlePortChange} type='number' />
+        <TextField
+          required
+          size='small'
+          value={formData.port}
+          onChange={handlePortChange}
+          error={formData.port !== '' && !isPortNumber(formData.port)}
+          helperText={formData.port !== '' && !isPortNumber(formData.port) ? 'Must be a valid port number' : ''}
```

Comment:
It would be useful to explain what a valid port number is. Perhaps change the helper text to `'Port number must be in the range 1-65535'`